tools/internal_ci/macos/grpc_basictests.cfg

```diff
@@ -15,7 +15,7 @@
 # Config file for the internal CI (in protobuf text format)
```

Comment:
Just saw you deleted `grpc_master.cfg` for linux in favor for `grpc_basictests.cfg`. `grpc_master.cfg` can be deleted in a future PR, after updated in Kokoro.